source/IAccessibleHandler.py

```diff
@@ -991,7 +991,7 @@ def getRecursiveTextFromIAccessibleTextObject(obj,startOffset=0,endOffset=-1):
 	except:
 		return text
 	textList=[]
-	for i in xrange(len(text)):
+	for i in range(len(text)):

```

Comment:
I think something like this should be considered for this case:

```suggestion
	for i, t in enumerate(text):
```